configs/shogun-sdk/Dockerfile

```diff
@@ -33,3 +33,8 @@ RUN cd /tmp;\
     cd arprec-2.2.18;\
     ./configure --enable-shared;\
     make install; ldconfig
+
+RUN wget www.scala-lang.org/files/archive/scala-2.11.7.deb
+RUN sudo dpkg -i scala-2.11.7.deb
+RUN apt-get update -qq
```

Comment:
there's absolutely no need to do update and apt-get install scala; the latter would actually would install trusty's scala 2.9...